### PR TITLE
change on On-Premise page

### PR DESCRIPTION
**Pull request by MagicLex**
Removed detailed insall and quick-start.

### CODE REVIEW

File: docs/setup_installation/on_prem/hopsworks_installer.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 
+It is possible to us Hopsworks on-premises, which means that companies can run their machine learning workloads on their own hardware and infrastructure, rather than relying on a cloud provider. This can provide greater flexibility, control, and cost savings, as well as enabling companies to meet specific compliance and security requirements.
 
-The _hopsworks-installer.sh_ script downloads, configures, and installs Hopsworks. It is typically run interactively, prompting the user about details of what to be is installed and where. It can also be run non-interactively (no user prompts) using the '-ni' switch.
+Working on-premises with Hopsworks typically involves collaboration with the Hopsworks engineering teams, as each infrastructure is unique and requires a tailored approach to deployment and configuration. The process begins with an assessment of the company's existing infrastructure and requirements, including network topology, security policies, and hardware specifications.
 
+For further details about on-premise installations; [contact us](https://www.hopsworks.ai/contact).
 
-## Requirements
+## Minimum Requirements
```
Comment (jimdowling) [inline docs/setup_installation/on_prem/hopsworks_installer.md:9]: ## Minimum Requirements

 * Ubuntu, Centos or RHEL Linux supported.

File: docs/setup_installation/on_prem/hopsworks_installer.md [hunk 2]
```diff
@@ -15,153 +17,10 @@ You need at least one server or virtual machine on which Hopsworks will be insta
 * outside Internet access (if this server is air-gapped, contact us for support),
 * a UNIX user account with sudo privileges.
 
+Further details about the on-premise installation of Hopsworks:
```
Comment (jimdowling) [inline docs/setup_installation/on_prem/hopsworks_installer.md:20]: remove this, no need for these details

File: docs/setup_installation/on_prem/hopsworks_installer.md [hunk 3]
```diff
@@ -15,153 +17,10 @@ You need at least one server or virtual machine on which Hopsworks will be insta
...
- [-ni|--non-interactive)] skip license/terms acceptance and all confirmation screens.
- [-p|--http-proxy) url] URL of the http(s) proxy server. Only https proxies with valid certs supported.
- [-hip|--hopsworks-public-ip) ip_address] Public IP address of the Hopsworks head VM.
- [-kip|--kafka-public-ip) ip_address] Public IP address of the Kafka broker (typically the head VM).
- [-pwd|--password password] sudo password for user running chef recipes.
- [-y|--yml yaml_file] yaml file to run Karamel against.
-
-```
\ No newline at end of file
+* If a multi-node installation is possible;
```
Comment (jimdowling) [inline docs/setup_installation/on_prem/hopsworks_installer.md:22]: remove all of these, no need for these details

**Review by jimdowling — CHANGES_REQUESTED** [review]

**Comment by SirOibaf** [comment]
Work moved here: https://github.com/logicalclocks/logicalclocks.github.io/pull/223

**Pull request closed without merging**